keras_cv/layers/preprocessing3d/group_points_by_bounding_boxes.py

```diff
@@ -231,16 +238,13 @@ def call(self, inputs, training=True):
                     ) = self.augment_point_clouds_bounding_boxes(
                         inputs[POINT_CLOUDS][i], inputs[BOUNDING_BOXES][i]
                     )
-                    object_point_clouds_list += [object_point_clouds[tf.newaxis, ...]]
-                    object_bounding_boxes_list += [
-                        object_bounding_boxes[tf.newaxis, ...]
-                    ]
-
+                    object_point_clouds_list += [object_point_clouds]
+                    object_bounding_boxes_list += [object_bounding_boxes]
                 inputs[OBJECT_POINT_CLOUDS] = tf.concat(
-                    object_point_clouds_list, axis=0
+                    object_point_clouds_list, axis=-3
```

Comment:
maybe comment on what -3 means here?
same for -2 below

Reply:
Added a comment.